File: core/java/src/main/java/com/vxlisp/vx/ui/svg/Uisvg.java
package com.vxlisp.vx.ui.svg;

import java.util.LinkedHashMap;
import java.util.Map;
import com.vxlisp.vx.*;
import com.vxlisp.vx.ui.*;

public final class Uisvg {


  static {
    Map<String, Core.Type_any> maptype = new LinkedHashMap<String, Core.Type_any>();
    Map<String, Core.Type_any> mapconst = new LinkedHashMap<String, Core.Type_any>();
    Map<String, Core.Type_func> mapfunc = new LinkedHashMap<String, Core.Type_func>();
    Core.vx_global_package_set("vx/ui/svg/uisvg", maptype, mapconst, mapfunc);
  }

}

File: core/kotlin/app/src/main/kotlin/com/vxlisp/vx/ui/svg/Uisvg.kt
package com.vxlisp.vx.ui.svg

import com.vxlisp.vx.*
import com.vxlisp.vx.ui.*

object vx_ui_svg_uisvg {


  init {
    var maptype : MutableMap<String, vx_core.Type_any> = LinkedHashMap<String, vx_core.Type_any>()
    var mapconst : MutableMap<String, vx_core.Type_any> = LinkedHashMap<String, vx_core.Type_any>()
    var mapfunc : MutableMap<String, vx_core.Type_func> = LinkedHashMap<String, vx_core.Type_func>()
    vx_core.vx_global_package_set("vx/ui/svg/uisvg", maptype, mapconst, mapfunc)
  }

}
